ui/src/components/executions/outputs/Wrapper.vue

```diff
@@ -432,6 +432,23 @@
         selectedValue.value !== debugExpression.value;
 
     const leftWidth = ref("70%");
+
+    const isMobile = ref(false);
+
+    onMounted(() => {
+        const mediaQuery = window.matchMedia("(max-width: 768px)");
+        isMobile.value = mediaQuery.matches;
+
+        const handler = (e: MediaQueryListEvent) => {
+            isMobile.value = e.matches;
+        };
+
+        mediaQuery.addEventListener("change", handler);
+
+        onUnmounted(() => {
+            mediaQuery.removeEventListener("change", handler);
+        });
+    });
```

Comment:
All composables should be used at the top of the setup function.

```suggestion
    });
    
     onUnmounted(() => {
          mediaQuery.removeEventListener("change", handler);
      });
```

This being said, I would prefer you use the `useMediaQuery()` of [vueuse](https://vueuse.org/core/useMediaQuery/) that will do all of that for you and more.

Reply:
I have made the required changes.